src/dmd/tokens.d

```diff
@@ -281,8 +271,7 @@ enum TOK : int
     ref_,
     macro_,
 
-    // 221
```

Comment:
This should have been `// 220`.  This mistake is no longer possible with this PR.